gateway/src/main/java/com/lyft/data/gateway/router/GatewayBackendManagerImpl.java

```diff
@@ -29,7 +31,11 @@ public GatewayBackendManagerImpl(List<ProxyBackendConfiguration> backends) {
           // We can make it agnostic of restart by persisting a copy of active backends to a
           // persistent cache.
           if (backend.isActive()) {
-            activeBackends.add(backend);
+            if (backend.isScheduledCluster()) {
```

Comment:
what happen when "two" clusters with same name, one is scheduled and the other is adhoc?